packages/libs/error-stack/src/ext/stream.rs

```diff
@@ -0,0 +1,360 @@
+//! Implements support for `error-stack` functionality for streams. The main trait in this module
+//! is [`StreamReportExt`] (which is so named to avoid conflict with other stream-related types
+//! common to the Rust `async` ecosystem).
```

Comment:
Could you add a new paragraph after the first sentence so rustdoc will show it up nicely in the overview?

Also, it might worth adding a link to `ResultExt` as this documentation is more elaborated.

Reply:
> Could you add a new paragraph after the first sentence so rustdoc will show it up nicely in the overview?

This was not addressed